src/app/api/stripe/[...path]/route.ts

```
@@ -0,0 +1,425 @@
+
+  return typeof stripeCustomerId === 'string' && stripeCustomerId.length > 0 ? stripeCustomerId : undefined;
+};
+
+const getPriceId = (interval: BillingInterval) => {
+  switch (interval) {
+    case 'monthly':
+      return runtimeEnv.stripe.priceIdProMonthly;
+    case 'yearly':
+      return runtimeEnv.stripe.priceIdProYearly;
+    default:
+      const _: never = interval;
```

Biome (2.4.10)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)